runtime/doc/insert.txt

```diff
@@ -1177,6 +1177,12 @@ items:
 	user_data	custom data which is associated with the item and
 			available in |v:completed_item|; it can be any type;
 			defaults to an empty string
+	hl_group	an additional highlight group whose attributes are
```

Comment:
since we added `kind_hlgroup` shortly after this, perhaps this should be named `hlgroup`

Reply:
maybe can change to abbr_hlgroup.